mysql-test/suite/rocksdb/t/blind_delete_without_tx_api.test

```diff
@@ -0,0 +1,131 @@
+--source include/have_rocksdb.inc
+
+source include/master-slave.inc;
+
+--source include/rpl_connection_master.inc
+
+set @save_rocksdb_blind_delete_primary_key=@@session.rocksdb_blind_delete_primary_key;
+set @save_rocksdb_master_skip_tx_api=@@session.rocksdb_master_skip_tx_api;
+
+--disable_warnings
+DROP TABLE IF EXISTS t1,t2;
```

Comment:
Ancient MTR idiom - no need to delete tables before creating them. MTR now always start from a clean state.